__tests__/app.test.js

```diff
@@ -142,4 +142,44 @@ describe("GET /api/articles",()=>{
             
         })
     })
+})
+
+describe("GET /api/articles/:article_id/comments",()=>{
+    it("200:should return with array of comments that match article id",()=>{
+        return request(app).get("/api/articles/1/comments").expect(200)
+        .then(({body})=>{
+
+            const commentArr = body
+            
```

Comment:
add a check for the length of the array here too since we are asserting inside a forEach